packages/jasmine/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "@bazel/jasmine",
-    "description": "Run Jasmine tests under Bazel",
+    "description": "Jasmine rules for Bazel",
+    "homepage": "https://github.com/bazelbuild/rules_nodejs",
```

Comment:
make it a deep link to /packages/jasmine?
I have the same addition in my PR

Reply:
I'll just take that commit out and we won't have a merge conflict :)